.github/workflows/multi-node.yml

```diff
@@ -36,8 +36,8 @@ jobs:
 
       - name: Setup Pods
         run: |
-          # Stress tests are using 13 nodes.
-          ./kubernetes/setup.sh 15 multi-node-test.hosts tcp
+          # Start 10 pods. At most 10 MultiJvmNode (akka.cluster.StressSpec is currently disabled).
+          ./kubernetes/setup.sh 10 multi-node-test.hosts tcp
```

Comment:
lol 

```
# we are using 13 nodes
$ use 15 nodes
```

Reply:
I guess the idea was 3 pods per node (we start 5 Kubernetes nodes)

Reply:
Aha, nodes vs pods